tokens.xml

```diff
@@ -1348,16 +1371,22 @@ Cloud Sprite can block only creatures with flying.</text>
             <reverse-related>Academy Manufactor</reverse-related>
             <reverse-related>Bearer of Overwhelming Truths</reverse-related>
             <reverse-related>Briarbridge Patrol</reverse-related>
+            <reverse-related>Briarbridge Tracker</reverse-related>
             <reverse-related>Bygone Bishop</reverse-related>
             <reverse-related>Byway Courier</reverse-related>
             <reverse-related>Confirm Suspicions</reverse-related>
             <reverse-related>Confront the Unknown</reverse-related>
             <reverse-related>Declaration in Stone</reverse-related>
+            <reverse-related>Dennick, Pious Apparition</reverse-related>
             <reverse-related>Drownyard Explorers</reverse-related>
+            <reverse-related>Eloise, Nephalia Sleuth</reverse-related>
+            <reverse-related count="x" exclude="exclude">Eloise, Nephalia Sleuth</reverse-related>
```

Comment:
1 is the default for x

Reply:
We have many other `count="x=1"` occurrences in the file though. So much mixed stuff :)

I guess it's better practice to include it in case something changes at Cockatice's end?

Reply:
For previous cases where it was used we have: "Maja, Bretagard Protector", Hullbreacher, "Malcolm, Keen-Eyed Navigator", Death Tyrant, From Under the Floorboards, Ayula's Influence, and probably more as well. This is not a new format for it.

Reply:
>This is not a new format for it. (Using a regular non-exclude for 1 token, and then an excluded x token relation.)

For my understanding those double entries were meant for cards that have two particular cases/options (not just a ranging x value), e.g. [Chancellor of the Forge](https://scryfall.com/card/nph/81/chancellor-of-the-forge):
- Create 1/1 Goblin
- Create X 1/1 Goblins

(He has no exclude btw... should that only be set for recurring token creators? 🤔)

Reply:
The reason I did it like this is because for the vast majority of cases they will only create one token, and we should have this as default for ease of use. The x option is just there for instances where they wont create one token.

Reply:
That makes sense @mainman879. I'm not questioning your reason, your intention was quite obvious when looking at the card. However, as ebbit noted, x defaults to 1 already.
There is still the benefit of a bit quicker token creation with one step less (no dialog). That can even be a good reason for double entries.

Don't get me wrong. I just try to find common rules that everybody is aware of to work together on this and maintain a good quality. Again, there are no guidelines yet.
At the end, it's the user who is annoyed that it works different for similar cards.